### PR TITLE
DD-1152 Invalid XML is also considered an ERROR

### PR DESCRIPTION
**Pull request by ericdevries**
Fixes DD-1152

# Description of changes

# How to test

# Related PRs
(Add links)
*

# Notify
@DANS-KNAW/dataversedans

### CODE REVIEW

File: src/test/java/nl/knaw/dans/validatedansbag/core/rules/XmlRulesImplTest.java [hunk 1]
```diff
@@ -102,6 +103,27 @@ void xmlFileDoesNotConformToSchema() throws Exception {
         assertEquals(RuleResult.Status.ERROR, result.getStatus());
     }
 
+    @Test
+    void xmlFileIsInvalid() throws Exception {
+
+        var xml = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n"
```
Comment (jo-pol) [inline src/test/java/nl/knaw/dans/validatedansbag/core/rules/XmlRulesImplTest.java:109]: I don't see where this variable is used.

Comment (ericdevries) [reply src/test/java/nl/knaw/dans/validatedansbag/core/rules/XmlRulesImplTest.java:109]: You are correct

Comment (janvanmansum) [reply src/test/java/nl/knaw/dans/validatedansbag/core/rules/XmlRulesImplTest.java:109]: @ericdevries : then shouldn't the variable be removed?

**Review by janvanmansum — CHANGES_REQUESTED** [review]
LGTM, tested it and worked as expected. @ericdevries : please, address the comment about the unused variable. Otherwise, ready for merge.

**Review by janvanmansum — APPROVED** [review]

**Pull request merged**